src/store/actions/postActions.js

```diff
@@ -1,154 +1,188 @@
+        },
+        { merge: true }
+      )
+      .then(() => {
+        dispatch({ type: "LIKE_SUCCESS", id });
+        return;
+      })
+      .catch((err) => {
+        dispatch({ type: "LIKE_ERROR", err });
+      });
+  };
+};
 
-        likes = likes.filter(function( obj ) {
-            return obj.uid !== uid;
-        });
+export const unlikePost = (id, uid, likes) => {
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+
+    likes = likes.filter(function (obj) {
+      return obj.uid !== uid;
+    });
+
+    // Create post
+    firestore
+      .collection("posts")
+      .doc(id)
+      .update({
+        likes: likes,
+      })
+      .then(() => {
+        dispatch({ type: "UNLIKE_SUCCESS", id });
+        return;
+      })
+      .catch((err) => {
+        dispatch({ type: "UNLIKE_ERROR", err });
+      });
+  };
+};
 
-        // Create post
-        firestore.collection('posts').doc(id).update({
-            likes: likes,
-        }).then(() => {
-            dispatch({ type: 'UNLIKE_SUCCESS', id });
-            return 
-        }).catch((err) => {
-            dispatch({ type: 'UNLIKE_ERROR', err });
+export const removePost = (uid, post) => {
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+
+    if (uid === post.data.author.uid) {
+      // Remove post
+      firestore
+        .collection("posts")
+        .doc(post.id)
+        .set(
+          {
+            visible: false,
+          },
+          { merge: true }
+        )
+        .then(() => {
+          dispatch({ type: "REMOVE_SUCCESS", id: post.id });
+          return;
         })
+        .catch((err) => {
+          dispatch({ type: "REMOVE_ERROR", err });
+        });
+    } else {
+      dispatch({ type: "REMOVE_ERROR", err: "Not authorized." });
     }
-}
+  };
+};
 
-export const removePost = (uid, post) => {
-    return (dispatch, getState, {getFirebase, getFirestore}) => {
-        const firebase = getFirebase();
-        const firestore = getFirestore();
-
-        if(uid === post.data.author.uid){
-            // Remove post
-            firestore.collection('posts').doc(post.id).set({
-                visible: false,
-            }, { merge: true }).then(() => {
-                dispatch({ type: 'REMOVE_SUCCESS', id: post.id });
-                return 
-            }).catch((err) => {
-                dispatch({ type: 'REMOVE_ERROR', err });
-            })
-        } else {
-            dispatch({ type: 'REMOVE_ERROR', err: "Not authorized." });
-        }
+export const loadPosts = (amount) => {
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+    dispatch({ type: "LOAD_LOADING" });
+
+    let posts = firestore.collection("posts").where("visible", "==", true);
```

Comment:
```suggestion
    let posts = firestore.collection("posts").where("visible", "==", true);
    
    dispatch({ type: "LOAD_LOADING" });

```
Is that possible? If so, adjust this. If not check this area for newlines.

Reply:
Nope